libraries/android/gradlew

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env sh
+
+# Attempt to set APP_HOME
+# Resolve links: $0 may be a link
+PRG="$0"
+# Need this for relative symlinks.
+while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+        PRG="$link"
+    else
+        PRG=`dirname "$PRG"`"/$link"
+    fi
+done
+SAVED="`pwd`"
+cd "`dirname \"$PRG\"`/" >/dev/null
+APP_HOME="`pwd -P`"
+cd "$SAVED" >/dev/null
+
+APP_NAME="Gradle"
```

Comment:
Shouldn't this be `Castor`?

Tbh I'm not sure what this file does.

Reply:
No, this is a generated runner.

TBH I also find it weird that it becomes part of the library/repository.

But I also find more things weird in Android... 😓